src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -79,6 +79,8 @@ public function getLocator()
      *
      * @throws FileLoaderLoadException
      * @throws FileLoaderImportCircularReferenceException
+     * @throws FileLocatorFileNotFoundException
+     * @throws \LogicException
```

Comment:
We only add `@throws` annotations when they are thrown directly by the method. Not sure why we have the 2 first one here. Anyway, `\LogicException` does not add any value here except if you add when this can be thrown.